internal/ui/component/watchlist/watchlist.go

```diff
@@ -210,13 +213,31 @@ func sortQuotes(q []quote.Quote) []quote.Quote {
 		}).
 		ResultAndError()
 
-	concatQuotes := gubrak.
-		From(activeQuotes).
-		OrderBy(func(v quote.Quote) float64 {
-			return v.ChangePercent
-		}, false).
-		Concat(inactiveQuotes).
+	quotesToShow := gubrak.
+		From(activeQuotes)
+
+	// Append the orderBy functionality
+	appendOrderBy(quotesToShow, sort, inactiveQuotes)
+
+	// Get the result from quotes
+	concatQuotes := quotesToShow.
 		Result()
 
 	return (concatQuotes).([]quote.Quote)
 }
+
+func appendOrderBy(quotes gubrak.IChainable, sort string, inactiveQuotes interface{}) {
+
+	switch strings.ToLower(sort) {
+	case "alpha":
+		quotes.Concat(inactiveQuotes)
```

Comment:
Should inactive quotes come before alphabetically sorted active? Also should active / inactive state of quotes be considered in sorting? I would personally expect when selecting alphabetical order that this would be applied independent of the market state but interested to hear the thinking behind this.

Reply:
The rationale behind this order is this:
1. sort by `alpha`:
the user would like to watch a **constant "view"** of the stocks
 (for instance, `ARKW` -> `BTC-USD` ->`QQQ` -> `SPY`), no mather what the state of the stock is.
_On lines 233,234 I first add the **inactiveQuotes** and than sort **all** the quotes_
2. sort by change percentage (default sorting)
the user would like to watch the change of the stocks during the day. all inactive quotes aren't so relevant, therefore are listed at the end of the view (maybe to sort the inactive stocks alphabetically in this case? 🤔 )



Based on what I write up there 👆🏼 I could change a little bit the code and I hope that it is what you meant:

<img width="514" alt="Screen Shot 2021-02-05 at 1 33 01" src="https://user-images.githubusercontent.com/2014537/106968902-7736ee80-6752-11eb-956a-7a1cbec62486.png">

Reply:
Thanks for your detailed response! That's totally my mistake - I misunderstood what was happening there on _233,234_